TransportClasses/Driver.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace TransportClass
+{
+    /// <summary>
+    /// Водитель
+    /// </summary>
+    public class Driver
+    {
+        /// <summary>
+        /// Индетификатор водителя
+        /// </summary>
+        public int ID { get; set; }
+
+        /// <summary>
+        /// ФИО
+        /// </summary>
+        public string FullName { get; set; }
```

Comment:
```suggestion
        public required string FullName { get; set; }
```
Если свойство (с ссылочным типом) не `nullable`, то надо либо задать `required`, либо написать

```csharp
 public string FullName { get; set; } = null!;
```

но лучше `required`
здесь и далее